src/screens/ManageFeedScreen.js

```diff
@@ -231,9 +155,15 @@ const mapStateToProps = ({ publication, settings }) => ({
 })
 
 export const ManageFeedScreen = connect(mapStateToProps)(ManageFeedScreenComp)
+
 ManageFeedScreen.navigationOptions = ({ route }) => ({
   title: 'Manage Feed',
   headerRight: () => (
-    <Button title={'Save'} onPress={() => route.params.handleSave()} />
+    <Button
+      title={'Save'}
```

Comment:
if a string is a prop, we don't need to wrap it with `{}`, just say `title="Save"` directly